src/dotnet/APIView/APIViewWeb/LeanControllers/APIRevisionsController.cs

```diff
@@ -155,8 +155,8 @@ public async Task<ActionResult<int>> GenerateAIReview(string reviewId, [FromQuer
         {
             try
             {
-                var violations = await _reviewManager.GenerateAIReview(User, reviewId, activeApiRevisionId, diffApiRevisionId);
-                return new LeanJsonResult(violations, StatusCodes.Status200OK);
+                await _reviewManager.GenerateAIReview(User, reviewId, activeApiRevisionId, diffApiRevisionId);
```

Comment:
is this the call that is taking a long time? do we need to await for it? speciallly if we are returning just the `Accepted`